src/test/java/org/redline_rpm/ant/RedlineTaskTest.java

```diff
@@ -90,15 +90,6 @@ public void testBadVersion() throws Exception {
 			// Pass
 		}
 
-		// test version with illegal char ~
-		task.setVersion("1.0~beta");
-		try {
```

Comment:
If we allow tilde, the current test fails. We could either change the test to assert the version is set successfully, or remove it. Since there would no longer be special handling for `~` with this PR, I opted to remove it -- but happy to make any changes.